cmd/csi-node-driver-registrar/node_register.go

```diff
@@ -20,22 +20,28 @@ import (
 	"fmt"
 	"net"
 	"os"
+	"os/signal"
 	"runtime"
+	"syscall"
 
 	"google.golang.org/grpc"
 
 	"k8s.io/klog"
 	registerapi "k8s.io/kubernetes/pkg/kubelet/apis/pluginregistration/v1alpha1"
 )
 
+const (
+	sockerDirPath = "/registration"
+)
+
 func nodeRegister(
 	csiDriverName string,
 ) {
 	// When kubeletRegistrationPath is specified then driver-registrar ONLY acts
 	// as gRPC server which replies to registration requests initiated by kubelet's
 	// pluginswatcher infrastructure. Node labeling is done by kubelet's csi code.
 	registrar := newRegistrationServer(csiDriverName, *kubeletRegistrationPath, supportedVersions)
-	socketPath := fmt.Sprintf("/registration/%s-reg.sock", csiDriverName)
+	socketPath := fmt.Sprintf("%s/%s-reg.sock", sockerDirPath, csiDriverName)
```

Comment:
nit: Could you pull this into a function like `buildSocketPath(csiDriverName)`? Right now, part of the socket path is pulled out as a const but part of it isn't

Reply:
updated now